server/src/test/java/org/elasticsearch/reservedstate/action/ReservedClusterSettingsActionTests.java

```diff
@@ -142,6 +143,6 @@ public void testSettingNameNormalization() throws Exception {
             """;
         TransformState newState2 = processJSON(testAction, prevState, jsonRemoval);
         assertThat(newState2.keys(), containsInAnyOrder("dummy.setting2"));
-        assertThat(newState2.state().metadata().persistentSettings().get("dummy.setting2"), equalTo("value2"));
+        assertThat(newState2.state().metadata().persistentSettings().get("dummy.setting2"), is("value2"));
```

Comment:
Why is this preferred?

Reply:
To match the others (where I've also used `is`)